cms/articles/tests/test_models.py

```diff
@@ -252,8 +242,8 @@ def test_clean_validates_a_max_of_six_headline_figures_can_be_added(self):
                 ),
                 "corrections": streamfield([]),
                 "notices": streamfield([]),
-                "headline_figures": streamfield([("figures", figures)]),
-                "headline_figures-0-value-count": 7,  # a bit of a hack to get the right nested format
+                "headline_figures": figures,
+                "headline_figures-count": 7,  # a bit of a hack to get the right nested format
```

Comment:
```suggestion
                "headline_figures": streamfield([
                    (
                        "figure",
                        {
                            "figure_id": f"figure_{i}",
                            "title": "Figure title XYZ",
                            "figure": "100 Million and more",
                            "supporting_text": "Reasons to add tests and use long test strings where possible",
                        }
                    )
                    for i in range(7)
                ]),
```

and `figures = [...]` can be removed

Reply:
Changed in [fc67459](https://github.com/ONSdigital/dis-wagtail/pull/181/commits/fc67459f307ac4cce61339808b2b0bf0ceb74cdc)